### ISSUE TITLE
Luminosity contrast ratio of Search and more options icon is less than 3:1.

### ISSUE THREAD

**Issue by msftedad** [telegramdesktop/tdesktop#27877]
### Steps to reproduce

1. Open Telegram App in Desktop.
2. Open URL: [Telegram: Contact @SCTejTestbot](https://t.me/SCTejTestbot) on edge.
3. SCTestbot home screen will appear.
4. Open CCA tool and verify the luminosity contrast ratio of 'Search and more options icon present in header section.
5. Observe an issue that Luminosity contrast ratio of Search and more options icon is less than 3:1.

### Expected behaviour

Luminosity contrast ratio of Search and more options should be greater than or equal to 3:1.

### Actual behaviour

Luminosity contrast ratio of Search and more options icon is less than 3:1.
<img width="621" alt="Luminosity contrast ratio of Search and more options icon is less" src="https://github.com/telegramdesktop/tdesktop/assets/93735775/64a27eaa-1f0d-48df-9842-283f9f2023d8">

### Operating system

22H2 (22621.1105)

### Version of Telegram Desktop

Telegram (4.16.4)

### Installation source

Microsoft Store

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/476
https://github.com/telegramdesktop/tdesktop/issues/2109
